.github/workflows/cd_debian.yml

```diff
@@ -0,0 +1,43 @@
+name: Debian Continuous Delivery
+
+on:
+  workflow_call:
+    inputs:
+      b2-upload-bucket:
+        required: true
+        type: string
+
+env:
+  PYTHON_DEFAULT_VERSION: "3.11"
+
+jobs:
+  build-deb:
+    env:
+      # dont run tests in Debian build, we ran them already
+      DEB_BUILD_OPTIONS: nocheck
+      B2_APPLICATION_KEY_ID: '${{ secrets.B2_DEBIAN_APPLICATION_KEY_ID }}'
+      B2_APPLICATION_KEY: '${{ secrets.B2_DEBIAN_APPLICATION_KEY }}'
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_DEFAULT_VERSION }}
+
+      - uses: jtdor/build-deb-action@v1
+        with:
+          buildpackage-opts: --build=binary
+
+      - uses: qualk/upload-artifact-as-is@master
+        with:
+          path: debian/artifacts/*.deb
+
+      - name: Upload to B2
+        working-directory: debian/artifacts
+        run: |
+          ls -1 *.deb | xargs -I {} sudo apt install ./{}
```

Comment:
is this a test? why are these packages being installed here?

Reply:
yes it's kind of a test